app/config/schema/config.proto

```diff
@@ -0,0 +1,216 @@
+/*
+ * This file defines and documents the format for config.json and other
+ * top-level configuration (base.json, development.json, production.json)
+ *
+ * Proto doc: https://developers.google.com/protocol-buffers/
+ */
+
+syntax = "proto3"; // Must be proto3 to support JSON
+
+message Config {
+    // TODO: What does this flag do?
+    bool debug = 1;
```

Comment:
@morganecf @hndrewaall What all is the debug flag for? I saw that it automatically turns on some of the navigation bar stuff (map etc), but does it do other stuff too?

Reply:
I think…it was intended to be more of a "developer mode" flag. I don't think it does anything else other than turn the map on automatically so that devs have access to it. It seems useful but I forgot it existed!

Reply:
I added an initial comment on this. We can adjust as needed.